### DISCUSSION TITLE
FORCE coregistration / base-images: sub-periods with multiple sets of base-images  (co-registration)

### DISCUSSION THREAD

**Discussion by Jay-Sig** [davidfrantz/force#111] [Q&A] [answered]
Hello,

I created NIR-base-images and according mosaics in order to  coregister some Sentinel-images. 
1) As I unterstand the FORCE-tutorial the single NIR-images of a base-image-set are an x-years-'tie-point-mean' for each month. --> the 'tie-point-mean' for January out of x years, the 'tie-point-mean' for February out of x years, ...? Right?
 If right: 
2) The FORCE-tutorial states, that you could also divide the total period (6 years in the example) in "multiple sets of base-images (e.g. for 2-year periods: 2015*, 2017*, 2019*)", instead of using a single base-image-set covering the total period.
So, I have to create 3 base-image-sets in 3 separate TSA-processes, i.e. the first one with date range from 20150101 to 20161231, the next one from 20170101 to 20181231,...? Right? Or did I overlook a one-step-option?
If right:
3) My trial with running _force-mosaic_ then resulted in 3 vrts (instead of one vrt as I expected). 
--> Is that right, and I do have accordingly to proceed the coregistration also in 3 separate two-year-subperiod-steps?  
--> Or, did I make any mistake, and the whole thing could be proceeded in only one step?

I just want be sure, to understand the principles right, as I am currently try to figure out what could be the best coregistration setup for my project. 

Regards, Jan

**Comment by davidfrantz** [comment, marked as answer]
Hi @Jay-Sig 

1. yes.
2.  correct.
3. this is correct as well. There should be one vrt for each period (each with 12 bands for the months). The preprocessing with force-level2 can be done in one step, the program will choose the correct base image.

Cheers,
David

**Reply by Jay-Sig** [reply]
Thanks, David again (also for your efforts with develloping force),
In case you are wondering about the reason for my questions, including the parallel post:
I did a coregistration trial with a section of my whole time series period. The congreuncy within the Sentinel-Images was significantly improved in comparison with the non-coreg-version (visual control).

Now, I had some TrueDOPs (with 40 cm georeferencing accuracy, according to the deliverer). Without coreg (of course) some Sentinels shifted towards east, some more west in relation to the TrueDOPs, and a few matched. **The average shift was between a half or one pixel** towards east (as far as this is verifiable by visual control). **But with coregistration the offset increased upto two pixels**, though, as said before, congruency within the Senitnels was much better.
So, I am about to figure out the origin of this error. Though I'm not absolutely sure, at the moment I rather think, there is something wrong with the TrueDOPs. 

Have a nice day

**Reply by davidfrantz** [reply]
Hi @Jay-Sig ,

we have to separate two things:

Multitemporal geolocation and absolute geolocation.
Please have a look at [Ferran Gascon's paper about the S2 product status](https://www.mdpi.com/2072-4292/9/6/584).

We mostly decrease the multitemporal geolocation uncertainty when applying a coregistration, which means that each image is "forced" to this baseline. Multitemporal geolocation is the key attribute to allow for accurate multi-temporal analyses

The absolute geolocation uncertainty can only be as good as the base image. Landsat Collection 1 has a global absolute accuracy of ~13m, which might explain parts of your issues. Consider using Landsat collection 2 (do you already use this?), which already used ESA's Global Reference Image, and improved the accuracy to ~6m. However, these values are global - they may differ from area to area. Absolute geolocation is the key attribute when you combine it with other data.

Cheers,
David

**Reply by Jay-Sig** [reply]
> Consider using Landsat collection 2 (do you already use this?)

I used _force-level1-csd_ for Download. My 'harvest' only consists of collection 1 (according to the fielname/metadata). I could/can not find an option to choose collection 2 in the tutorial. Is there any? Or would I have to download Coll2-images 'manually' (i.e. without using FORCE)?

**Reply by ernstste** [reply]
force-level1-csd (CloudStorageDownloader) pulls data from Google Cloud Storage. Landsat Collection 2 data is unfortunately not available on GCS at this point. 
You can use the USGS bulk downloader or create downloadlinks with [landsatlinks](https://github.com/ernstste/landsatlinks) and use a download manager of your choice.
The fastest option would be downloading from AWS s3 requester pays buckets - if paying is an option, that is.

Stefan
